bin/helpers.js

```diff
@@ -3,8 +3,8 @@
 const Configstore = require('configstore');
 const conf = new Configstore('napp-config');
 
-const changeAllSettings = (bool) => {
-    conf.set({
+const changeAllSettings = (store=conf, bool) => {
```

Comment:
Just curious, but why did you change the name to store here?  Just to be more verbose or was there a specific reason?

Reply:
I added that `store` parameter so that I could pass in a separate `test-napp-config` store for testing purposes. It helps avoid manipulating our local `nap-config` stores.

Reply:
Oh, I gotcha!  Thanks for letting me know!